hoomd/md/external.py

```diff
@@ -5,9 +5,9 @@
 
 R""" External forces.
```

Comment:
Replace this with just 
```"""External forces for molecular dynamics."""

Reply:
I don't understand what this is supposed to to mean. Are you saying get rid of the line that says External forces, or are you saying get rid of the empty line below that one?

Reply:
Make it consistent with the other packages that use just a one line description, like
```
"""Pair potentials."""
```
in `pair.py`.

There is no need for tutorial-like descriptions in the reference documentation. In any case `Integrator` documents the types of objects that can be added to the forces list. Check the `Integrator` docs to make sure that the external module is mentioned.